src/components/Practice/Mechanics6.jsx

```
@@ -0,0 +1,518 @@
+      audioRef.current?.pause();
+      setIsPlaying(false);
+    } else {
+      audioRef.current.pause();
+      audioRef.current.load();
+      audioRef.current.play();
+      setIsPlaying(true);
+    }
+  };
+
+  const [currrentProgress, setCurrrentProgress] = React.useState(0);
+  const progressBarWidth = isNaN(currrentProgress / duration)
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)